synapse/lib/types.py

```diff
@@ -625,6 +625,9 @@ def _storLiftPref(self, cmpr, valu):
         )
 
     def _normPyList(self, valu):
+        if len(valu) == 0:
```

Comment:
```suggestion
        if not valu:
```

The __bool__ for a list or tuple is the bool of its length; so we can get that directly without the call + compare.